astronomy-pipeline/astronomy_transform_functions.py

```diff
@@ -0,0 +1,85 @@
+"""Functions for the astronomy API transform script."""
+
+import json
+import pandas as pd
+
+from astronomy_extract_functions import get_db_connection
+
+
+def load_from_file(filename: str) -> dict:
+    """Load the stories from a file called stories.json"""
+
+    with open(filename, "r", encoding="UTF-8") as f_obj:
+        return json.load(f_obj)
+
+
+def get_data_into_database(raw_data: dict) -> pd.DataFrame:
```

Comment:
Should this say `get_data_into_dataframe`?

Reply:
Yes good spot.